packages/collector/test/tracing/frameworks/node-fetch/app.js

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Q: Putting this here. node-fetch is not a framework.  How about putting it to "misc"?

Reply:
Moved to misc